src/aap_eda/settings/default.py

```diff
@@ -198,6 +199,16 @@ def _get_boolean(name: str, default=False) -> bool:
     "aap_eda.core",
 ]
 
+FLAGS = {
+    "EDA_ANALYTICS": [
+        {
+            "condition": "boolean",
+            "value": settings.get("FEATURE_FLAG_EDA_ANALYTICS", True),
```

Comment:
We already have the automatic "EDA" prefix, so the environment variable will be `EDA_FEATURE_FLAG_EDA_ANALYTICS` which seems a bit redundant. 
There is no chance of collision, so I would just use `FEATURE_FLAG_ANALYTICS`

Also, I don't know if we already agreed a name pattern, but I would like to use the "ENABLE" term to be clearer.
I propose `ENABLE_FEATURE` prefix, "ENABLE_FEATURE_[feature name]" with "ENABLE" I feel we state clearly what means the value and that we expect a boolean. 

 If FEATURE_FLAG has been already decided for the whole platform, it's fine then.

Reply:
It has not yet been decided for the whole platform, but I think that is a great note.

I can change this variable name to - `ENABLE_FEATURE_EDA_ANALYTICS`.

Since this is intended to be eventually toggled on/off at the platform level, and it will not affect other components analytics, I think it is good to keep the `EDA` part in there currently, even though this may be redundant.

Reply:
I ended up renaming this too - `FEATURE_EDA_ANALYTICS`, with the exception of the `EDA_` leading prefix, I figured it may be good to not set a standard on starting this variable with `ENABLE` for now.
I think it would be good to try to ensure all feature flag variables start with `FEATURE_` (ignoring the potential component leading prefix)